packages/cli/src/commands/import.ts

```diff
@@ -0,0 +1,60 @@
+import { Command, flags } from '@oclif/command';
+import { promises as fs } from 'fs';
+import { Messages } from '../constants/messages.constants';
+import { parseDataFiles } from '../libs/data';
+
+export default class Import extends Command {
+  public static description = 'Import a mock API';
+  public static examples = [
+    '$ mockoon-cli import --data ~/data.json --output ./output.json',
+    '$ mockoon-cli import --data ~/data.json --output ./output.json --type open-api-v3'
+  ];
+
+  public static flags = {
+    data: flags.string({
+      char: 'd',
+      description: 'Path or URL to your data file',
+      required: true
+    }),
+    output: flags.string({
+      char: 'o',
+      description:
+        'Generated Mockoon path and name (e.g. `./environment.json`)',
+      required: true
+    }),
```

Comment:
I find these flags a bit confusing as `--data` is for Mockoon's file in all other commands (including the new export). 
Here, it's for the OpenAPI source file. 
I would rather use `--data` as the output and `--input` as the source. 
What do you think?

Reply:
I also think this data flag a bit confusing at first.
We can follow the pattern of other applications with input and output flags.
IMO it would be better to keep the output instead of data.
Let me know your thoughts.